contract/nft/nft.go

```diff
@@ -0,0 +1,80 @@
+package nft
+
+import (
+	"context"
+	"errors"
+	"github.com/tonkeeper/tongo"
+	"github.com/tonkeeper/tongo/abi"
+	"github.com/tonkeeper/tongo/boc"
+	"github.com/tonkeeper/tongo/tlb"
+	"github.com/tonkeeper/tongo/wallet"
+	"math/big"
+	"time"
+)
+
+type Item struct {
+	Address    tongo.AccountID
+	Collection *tongo.AccountID
+	Owner      *tongo.AccountID
+}
+
+type sender interface {
+	Send(context.Context, ...wallet.Sendable) error
+	GetAddress() tongo.AccountID
+}
+
+func (item Item) Send(ctx context.Context, sender sender, destination tongo.AccountID) error {
```

Comment:
```golang
// we ask an item to send itself
item.Send(wallet)
//vs
// we ask a wallet to send an item
wallet.Send(item)
```
idk, personally, I like the second approach more because it sounds more natural to ask a contract to send an item.

Reply:
Wallet already have method Send, but it receives `ItemTransferMessage` not `Item`. But we can rename NFT method to `Transfer`  which is more appropriate.